lib/item.rb

```diff
@@ -28,18 +33,34 @@ def update_id(id)
   end
 
   def update_name(name)
-    @name = name
+    @name = name unless name.nil?
   end
 
   def update_description(description)
-    @description = description
+    @description = description unless description.nil?
   end
 
   def update_unit_price(unit_price)
-    @unit_price = unit_price
+    @unit_price = unit_price unless unit_price.nil?
   end
 
   def update_time
     @updated_at = Time.now
   end
+
+  def created_at
+    if @created_at.instance_of?(Time)
+      @created_at
```

Comment:
Ah I see the work you're having to do here to get the time methods to agree with the spec harness. Maybe the element I'm overlooking in later classes is how they interact when reading more complex data from the actual CSV? Mine pass the tests currently but I'll remember this if anything goes strange for CSV reading or spec harness. Looks nice.

Reply:
Yea, the spec harness sends stuff to you in weird formats, so I think it's pushing us to account for different inputs.